src/main.rs

```diff
@@ -14,10 +14,8 @@ fn main() {
         File::open("src/question.json").expect("Failed to reading file: question.json.");
     let subjects: Vec<Subject> = serde_json::from_reader(questions_file).unwrap();
 
-    let mut count = 0;
     let length = subjects.len();
-    for subject in &subjects {
-        count += 1;
+    for (count, subject) in subjects.iter().enumerate() {
```

Comment:
Oops... @lili668668 I forgot to update this 😭 It's now like `0/9`. Let me send another PR, sorry 🙏